ReleaseChecklist.md

```diff
@@ -62,7 +63,10 @@
  - [ ] Create a tag using ``git tag --annotate v$VERSION`` and push it with ``git push --tags``.
 
 ### Post-release
- - [ ] Publish the blog post.
+ - [ ] Make sure the documentation for the new release has been published successfully.
+       Go to https://readthedocs.org/projects/solidity/ and verify that the new version is listed, works and is marked as default.
+ - [ ] Remove "still in progress" warning from the release page.
+ - [ ] Publish the blog posts.
  - [ ] Create a commit to increase the version number on ``develop`` in ``CMakeLists.txt`` and add a new skeleton changelog entry.
  - [ ] Announce on Twitter, including links to the release and the blog post.
  - [ ] Share announcement on Reddit and Solidity forum.
```

Comment:
```suggestion
 - [ ] Share announcement on Reddit (r/ethdev) and Solidity forum at forum.soliditylang.org in the "Announcements" category.
```

Reply:
What about `/r/ethereum`?

Reply:
yep... or crossposting

Reply:
ok, I added a note about cross-posting, I can still remove it if you think `/r/ethdev` is enough.

I also split forum and reddit into separate points.